config.go

```diff
@@ -182,57 +170,30 @@ func (c *wafConfig) clone() *wafConfig {
 	return &ret
 }
 
-type requestBodyConfig struct {
-	limit         int
-	inMemoryLimit int
-}
-
-func (c *requestBodyConfig) WithLimit(limit int) RequestBodyConfig {
-	ret := c.clone()
-	ret.limit = limit
-	return ret
-}
-
-func (c *requestBodyConfig) WithInMemoryLimit(limit int) RequestBodyConfig {
+func (c *wafConfig) WithRequestBodyLimit(limit int) WAFConfig {
 	ret := c.clone()
-	ret.inMemoryLimit = limit
+	ret.requestBodyLimit = limit
 	return ret
 }
 
-func (c *requestBodyConfig) clone() *requestBodyConfig {
-	ret := *c // copy
-	return &ret
-}
-
-type responseBodyConfig struct {
-	limit         int
-	inMemoryLimit int
```

Comment:
responseBodyConfig had `inMemoryLimit`, but then it was not populated nor the `WAF struct` has a corresponding default value and field for it. It is okay to assume that it was not used, and not needed?